docs/conf.py

```diff
@@ -0,0 +1,64 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "dspace-python-client"
+copyright = "2021, Helen Bailey, Eric Hanson"
```

Comment:
I usually put MIT Libraries as the copyright owner in the license files I create. No idea if that is correct but I'm pretty sure we don't retain copyright on works for hire without an explicit contract stating we do (which we don't have).

Reply:
Oh good catch, thank you! Will update.